### ISSUE TITLE
Security problem

### ISSUE THREAD

**Issue by michelhaik** [IMCreatorLTD/QA-Testing#4]
![image](https://cloud.githubusercontent.com/assets/18535767/24657298/5841b236-194e-11e7-9ea6-e74d21e8dc20.png)
If i choose an premium template, the website don't have a security adress (aka HTTPS), so the credit card data is send by a non security way

**Comment by boozi2** [comment]
That's True, the main site is not via https but the small buying window is an iframe with https in it
